src/Symfony/Component/Uid/Command/GenerateUuidCommand.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Uid\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\ConsoleOutputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Uid\Uuid;
+
+class GenerateUuidCommand extends Command
+{
+    protected static $defaultName = 'uuid:generate';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure(): void
+    {
+        $this
+            ->setDefinition([
+                new InputArgument('version', InputArgument::OPTIONAL, 'The UUID version to generate (v1, v3, v4, v5 or v6).', 'v1'),
```

Comment:
should it default to v4 instead?
or do we make the argument mandatory? Then, we could rename the command `uid:generate`, and make it handle ulid and uuids?

Reply:
I would indeed make it required

Reply:
@fancyweb https://github.com/symfony/symfony/pull/36405 did so, maybe you can borrow some idea from there?

Reply:
I agree that we can make it required but I would still vote personally to separate the uuid and ulid.